document/src/main/java/com/ritense/document/web/rest/impl/SearchFieldResource.java

```diff
@@ -53,4 +54,15 @@ public ResponseEntity<List<SearchField>> getSearchField(
         @PathVariable String documentDefinitionName) {
         return ResponseEntity.ok(searchFieldService.getSearchFields(documentDefinitionName));
     }
+
+    @Override
+    @PutMapping("/v1/document-search/{documentDefinitionName}/fields")
+    public ResponseEntity<List<SearchField>> updateSearchField(
+            @RequestBody SearchField searchField) {
+        if(searchField.getId() == null){
```

Comment:
Thet frontend might not send back the 'id' property. Because the frontend isn't even supposed to rget the 'id'. They work with the 'key' property

Reply:
yeah i thought of that but i wasn't sure. will fix

Reply:
fixed